src/transformers/modeling_bert.py

```diff
@@ -310,7 +332,7 @@ def forward(self, hidden_states, input_tensor) -> torch.Tensor :
 class BertScriptableAttention(nn.Module):
     def __init__(self, config):
         super().__init__()
-        self.self = BertScriptableSelfAttention(config)
+        self.self =  BertScriptableSelfAttention(config)
```

Comment:
extra space?

Reply:
Yes, couldn't run the styling tools with the branch half there half not-there.